### ISSUE TITLE
Support query with special analyzer? 

### ISSUE THREAD

**Issue by siddontang** [blevesearch/bleve#189]
Hi

I know that Elastcisearch can support index and search using different analyzer. I browse bleve's code and find that it only uses the defined analyzer in mapping. Am I wrong?

This may be an excellent feature for Chinese search. 

Btw, I find that bleve has no good Chinese analyzer too. :-) Any plan?

**Comment by mschoch** [comment]
You are correct, at the moment we only let you configure a single analyzer for a field, and we use it for both index and search.  I don't anticipate this being a huge change, its mostly just adding some additional configuration and rules to follow it.  Mainly we ignored it at first since for lots of cases you use the same analyzer.

Regarding specific analyzers for Chinese, there is another issue open for that.  See #128 

In that issue people have linked to what look to be 2 different analyzers for Chinese.  I can make an educated guess on how to use them from the API, but the documentation is in Chinese.  It would certainly be easier if a native speaker helped us integrate one of these (or both if appropriate).

**Comment by mschoch** [comment]
Here are two pages describing how different search and index analyzers are used in Elasticsearch:

http://stackoverflow.com/questions/15923480/elastic-search-search-analyzer-vs-index-analyzer
http://www.elastic.co/guide/en/elasticsearch/guide/current/_controlling_analysis.html

**Comment by gg4u** [comment]
I m looking at examples about how to use search_analyzer and index_analyzer:
[https://gist.github.com/alinpopa/1281448]

How are they called?
I don't see the `analyzer` parameter called in `mapping` settings.
instead here [https://gist.githubusercontent.com/justinvw/5025854/raw/c278e4c76351b7f82834ed1f3de47a66365cbfd0/es_simple_autocomplete_example_config.sh] and in documentation I see it.

Could you please mockup an example reflecting the  answer:
[http://stackoverflow.com/a/15932838/305883] ?
